banyan-cli/src/tag_index.rs

```diff
@@ -23,6 +28,29 @@ pub struct TagIndex {
     pub(crate) elements: Vec<IndexSet>,
 }
 
+impl Encode<DagCborCodec> for TagIndex {
+    fn encode<W: std::io::Write>(&self, c: DagCborCodec, w: &mut W) -> anyhow::Result<()> {
+        todo!()
```

Comment:
ready for review? :)

Reply:
oh well, you are right. I was thinking about getting rid of the whole banyan-cli thing and instead having an ax trees cli inside cosmos.

Reply:
Implemented it in a very inefficient way for now. I might add libipld support to my vec_collections in the longer term.